runtimes/deno-1.24/src/server.ts

```diff
@@ -58,6 +58,11 @@ const action = async (logger: Logger, ctx: any) => {
     headers[header.toLowerCase()] = ctx.request.headers.get(header);
   });
 
+  const enforcedHeaders = JSON.parse(Deno.env.get("OPEN_RUNTIMES_HEADERS") ? (Deno.env.get("OPEN_RUNTIMES_HEADERS") ?? '{}') : '{}');
```

Comment:
```suggestion
  const enforcedHeaders = JSON.parse(Deno.env.get("OPEN_RUNTIMES_HEADERS") ?? '{}');
```
why not just this?

Reply:
Same as https://github.com/open-runtimes/open-runtimes/pull/248#discussion_r1638043411 , sadly it wont work for use-case with ENV variables